src/types/Posts.ts

```diff
@@ -0,0 +1,6 @@
+export interface Posts {
```

Comment:
The interface name should be singular since it represents a single post. Consider changing it to 'Post'